Some more documentation on the headless mode. (#94)
ilastik/ilastik.github.io: documentation/basics/headless.md

- `--output_axis_order` Transpose the storage order of the results. For example, this affects the sliced dimension for stack outputs.
- `--pipeline_result_drange` Pipeline result data range (min,max) BEFORE normalization, e.g. `"(0.0,1.0)"`
- `--export_drange` Exported data range (min,max) AFTER normalization, e.g. `"(0,255)"`

> - `--readOnly=1` Open project in read-only mode to allow multiple processing accessing the same file.

Reviewer: ```suggestion
- `--readonly` Open the project in read-only mode; necessary for using a single project from multiple processes.
```

    # Mac
    $ ./ilastik-1.3.2-OSX.app/Contents/ilastik-release/bin/python -c "import ilastik; print ilastik.__version__"
    1.3.2

> Check out [this example](https://github.com/ilastik/ilastik/blob/master/examples/example_python_client.py) on how to use ilastik as a python module

Reviewer 2: We plan on changing the way to use ilastik as a python module completely and don't want people to start building projects now, relying on the old way.
```suggestion
```

Author: Oh I'm sorry I didn't know that. I am currently building a project relying on that and it took some time for me to find this example.
